graviti/portex/factory.py

```diff
@@ -616,8 +640,12 @@ def factory_creator(
         A ``Factory`` instance according the input.
 
     """
-    if isinstance(decl, str) and decl.startswith("$params."):
-        return VariableFactory(decl[8:], ptype)
+    if isinstance(decl, str):
+        if decl.startswith("$params."):
+            return VariableFactory(decl[8:], ptype)
+
+        if decl.startswith("+$params."):
```

Comment:
`TypeFactory` and `DictFactory` also use `factory_creator`, what if someone use `"+$params."` in `TypeFactory` and `DictFactory`?